client/src/layouts/MainLayout.vue

```diff
@@ -23,6 +23,17 @@
               {{ $t("header.dashboard") }}
             </q-item-section>
           </q-item>
+          <q-item
+            v-ripple
+            to="/account/profile"
+          >
+            <q-item-section avatar>
+              <q-icon name="account_circle" />
+            </q-item-section>
+            <q-item-section>
+              {{ $t("header.account_link") }}
+            </q-item-section>
+          </q-item>
```

Comment:
I don't have a problem with this change, just curious if it was an errant inclusion since everything else in this PR is API only,

Reply:
Yes, I mentioned this in the PR description.

Reply:
> Bonus: adds a link to the Account Profile page to the sidebar menu on the frontend.

Reply:
Oops sorry! I did read it but must have completely forgotten by the time I got to commenting.